pglast/enums/__init__.py

```diff
@@ -12,3 +12,32 @@
 from .primnodes import *    # noqa
 from .nodes import *        # noqa
 from .parsenodes import *   # noqa
+
+
+class GroupingSetKind(IntEnum):
+    GOUPING_SET_EMPTY = 0,
```

Comment:
Is the trailing comma intended or is it a typo?
```suggestion
    GOUPING_SET_EMPTY = 0
```

Reply:
Typo, good catch :)